src/fxp/Repository/Vcs/AbstractGitHubDriver.php

```
@@ -0,0 +1,229 @@
+    }
+
+    /**
+     * Push the list of all branch.
+     *
+     * @param string $resource
+     * @param array  $branchBlacklist
+     */
+    protected function doAddBranches($resource, array $branchBlacklist)
+    {
+        do {
+            $branchData = JsonFile::parseJson((string) $this->getContents($resource), $resource);
```

PHPStan (2.2.8)

[error] Cannot cast Composer\Util\Http\Response to string.


(cast.string)